support/kind/kind.go

```diff
@@ -46,6 +46,51 @@ func (k *Cluster) WithVersion(ver string) *Cluster {
 	return k
 }
 
+func (k *Cluster) CreateWithParam(imageName string, kindConfigFile string) (string, error) {
```

Comment:
Since this is taking a kind config file, would a more descriptive name like `CreateWithConfig` be a better name?